en/docs/install-and-setup/sso/configure-identity-server-as-external-idp-using-saml.md

```diff
@@ -0,0 +1,262 @@
+            </tr>
+            <tr>
+                <td>Enable Signature Validation in Authentication Requests and Logout Requests</td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td>Enable Single Logout</td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td>Enable Attribute Profile</td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td>Include Attributes in the Response Always</td>
+                <td>true</td>
+            </tr>
+        </tbody>
+        </table>
+
+        The following image shows the sample values for SAML2 Web SSO configuration:
+
+        [![SAML configuration in Service Provider]({{base_path}}/assets/img/setup-and-install/saml-configuration-in-service-provider.png)]({{base_path}}/assets/img/setup-and-install/saml-configuration-in-service-provider.png)
+
+        ??? Info "Enable a tenant-specific SSO for the Publisher and Developer Portal"
+
+            To enable a tenant-specific SSO with IS 5.9.0 for Publisher and the Developer Portal, enable the **Use tenant domain in local subject identifier** option under the **Local & Outbound Authentication Configuration** section.
+
+            [![saml-configuration-in-service-provider]({{base_path}}/assets/img/setup-and-install/enable-tenant-domain-in-local-sub-identifier.png)]({{base_path}}/assets/img/setup-and-install/enable-tenant-domain-in-local-sub-identifier.png)
+
+    2.  Expand the **Claim Configuration** section and add **http://wso2.org/claims/role** as a mandatory claim.
+
+        [![Claim configuration in Service Provider for SAML2 SSO]({{base_path}}/assets/img/setup-and-install/claim-configuration-in-service-provider-for-saml2-sso.png)]({{base_path}}/assets/img/setup-and-install/claim-configuration-in-service-provider-for-saml2-sso.png)
+
+    3.  Update the Service Provider configurations.
+
+        !!! Info "In multi-tenanted environments"
+            Carry out the instruction given below for all the tenants to be able to sign in to the API-M Web applications in a multi-tenanted environment.
+
+            1. Click the **SaaS Application** option that appears after registering the Service Provider.
+
+               [![saas-configuration-in-service-provider]({{base_path}}/assets/img/setup-and-install/saas.png)]({{base_path}}/assets/img/setup-and-install/saas.png)            
+
+            If you do not select the **SaaS Application** option, only users in the current tenant domain will be allowed to sign in to the portals. You will need to register separate Service Providers for portals from each tenant.
+
+4.  Upload the public certificate of the API Manager by selecting **Select SP Certificate Type**.
+
+    [![Upload certificate in SP for SAML2 SSO]({{base_path}}/assets/img/setup-and-install/upload-certificate-in-SP-for-saml2-sso.png)]({{base_path}}/assets/img/setup-and-install/upload-certificate-in-SP-for-saml2-sso.png)
```

Comment:
Rename "upload-certificate-in-SP-for-saml2-sso.png" as capital letters should not be added in images and file names.

Reply:
Created action item [1] for this issue.

[1] https://github.com/wso2/docs-apim/issues/2195